diff --git a/src/docGeneration/generate.py b/src/docGeneration/generate.py
index a4b211ea3b..26b4987e13 100755
--- a/src/docGeneration/generate.py
+++ b/src/docGeneration/generate.py
@@ -33,7 +33,7 @@ for doc_file in documentation_files:
         .replace(".rs.md", ".rs")
     )
     corresponding_source_file = "src/" + doc_file.split("/", 2)[2]
-    
+
     if not os.path.exists(corresponding_source_file):
         print(f"Removing outdated documentation: {doc_file}")
         os.remove(doc_file)
diff --git a/src/trigSolver/main.py b/src/trigSolver/main.py
index d6498ed4b7..bc4f844154 100644
--- a/src/trigSolver/main.py
+++ b/src/trigSolver/main.py
@@ -7,6 +7,7 @@ opposite_input = input("Enter the length of the opposite side: ")
 adjacent_input = input("Enter the length of the adjacent side: ")
 angle_input = input("Enter the angle in degrees: ")
 
+
 def parse_input(input_str):
     if input_str == "":
         return None
@@ -15,6 +16,7 @@ def parse_input(input_str):
     except ValueError:
         return None
 
+
 hypotenuse = parse_input(hypotenuse_input)
 opposite = parse_input(opposite_input)
 adjacent = parse_input(adjacent_input)
@@ -55,4 +57,4 @@ if adjacent is None:
 print("The adjacent side is:", adjacent)
 print("The opposite side is:", opposite)
 print("The hypotenuse is:", hypotenuse)
-print("The angle is:", angle)
\ No newline at end of file
+print("The angle is:", angle)
